### DISCUSSION TITLE
简报: 新版动态页面的灰度范围正在进一步扩大

### DISCUSSION THREAD

**Discussion by Tinhone** [the1812/Bilibili-Evolved#3999] [综合讨论 / General]
如题，这是我的情况：
正常打开 t.bilibili.com 不会转至新版动态页面 (`https://t.bilibili.com/?w=3`)。不过，如果一条动态是转发动态，点击被转发动态的 `详情 >` 便能进入新版动态页面，即 https://www.bilibili.com/opus/*

**Comment by timongh** [comment]
我今天被灰度到了。不需要 w=3 直接进

小分辨率：
<img width="960" alt="little" src="https://user-images.githubusercontent.com/46739861/220607282-36c5d6b7-7dbe-45b3-93c7-265afa4964bd.png">

大分辨率
<img width="960" alt="big" src="https://user-images.githubusercontent.com/46739861/220607325-8861270a-092a-44b4-aab9-f5a47f289916.png">

**Comment by Tinhone** [comment]
#3999

**Comment by Tinhone** [comment]
![图片](https://user-images.githubusercontent.com/81268244/222375100-31e47917-6efb-45ff-91cd-0d85be54ca65.png)
https://www.bilibili.com/opus/710506024665088114
